tests/unit/services/LibraryService.test.js

```diff
@@ -55,4 +55,16 @@ describe('Test: LibraryService', () => {
       expect(api.delete).toBeCalledWith('/api/classes/Library/test', { headers: undefined });
     });
   });
+
+  describe('Test function: create', () => {
+    it('should create the library corresponding', async () => {
```

Comment:
should create the corresponding library (but the test does not check correspondence)
OR
should return the created library (only return the `objectId`)

Reply:
Think is done, from justine comment.